### DISCUSSION TITLE
Default Pool Size

### DISCUSSION THREAD

**Discussion by penguine-ip** [supabase/supabase#20573] [Questions]
Hey all, currently in supabase the default pool size is 15. However, I am running a high load serverless architecture with vercel, next.js, and prisma stack, and if i receive >15 rapid requests at the same time, it errors. Is there a way to work around this? I'm not sure is it a matter of handling things in code, or is there a supabase configuration for me to work with. 

Thanks!

**Comment by GaryAustin1** [comment]
![image](https://github.com/supabase/supabase/assets/54564956/e989ac9b-2678-4052-8db1-379cdb004ac4)

You can change it.  I would follow the links from there to see if there is more info.
